### PR TITLE
Updating zk quorum tests

### PR DESCRIPTION
**Pull request by amitaekbote**
Updating zk quorum check to include tests, also fixed bugs that the tests caught

### CODE REVIEW

File: cmd/checks.go [hunk 1]
```diff
@@ -13,6 +13,9 @@ const (
 	statusWarning
 	statusFailure
 	statusUnknown
+
+	// exhibitor admin router port
+	exhibitorPort = 8181
```
Comment (initcore0) [inline cmd/checks.go:18]: maybe we should make it configurable from CLI and set default value to 8181?

File: cmd/zk_quorum_check.go [hunk 2]
```diff
@@ -33,9 +33,9 @@ const zkResponseServing = 3
 
 // NewZkQuorumCheck returns an initialized instance of *ComponentCheck.
 func NewZkQuorumCheck(name string) DCOSChecker {
-	return &ZkQuorumCheck{
-		Name: name,
-	}
+	check := &ZkQuorumCheck{Name: name}
+	check.urlFunc = check.getURL
```
Comment (initcore0) [inline cmd/zk_quorum_check.go:37]: hm, what are you trying to do here?

**Review by initcore0 — CHANGES_REQUESTED** [review]

File: cmd/zk_quorum_check.go (hunk 2 above)
Comment (amitaekbote) [reply cmd/zk_quorum_check.go:37]: This is what we did for being able to test the json parsing with a random url, if you look at the httptest we needed this. more discussion https://github.com/dcos/dcos-checks/pull/5

File: cmd/checks.go (hunk 1 above)
Comment (amitaekbote) [reply cmd/checks.go:18]: Yup we will move all these constants to the runner once https://github.com/dcos/dcos-checks/pull/9 gets merged

**Review by initcore0 — APPROVED** [review]

**Pull request merged**